Dockerfile.dev

```diff
@@ -12,7 +12,7 @@ WORKDIR /usr/src/app
 
 RUN gem install bundler
 COPY Gemfile Gemfile.lock ./
-RUN bundle install
+RUN bundle install --full-index
```

Comment:
According to my Google-fu, this is a way to speed up the install. Should I also add this to the other dev Dockerfiles?

Reply:
> According to my Google-fu, this is a way to speed up the install. Should I also add this to the other dev Dockerfiles?

So it actually slows down the install! What it does do though is run against the full Rubygems index, rather than a cache. The reason I'm doing this is because I'm pushing the `exercism_config` gem to Rubygems, but it takes 10-15mins to hit a cache, so in that space, nothing can build. This forces it to read from the real index.

> Should I also add this to the other dev Dockerfiles?

It feels like a sensible thing to have everywhere, yeah!

Reply:
Ah, I read it as "download in one request" instead of multiple :D